src/storage/txn/commands.rs

```diff
@@ -99,6 +251,61 @@ impl Prewrite {
             }),
         }
     }
+
+    #[cfg(test)]
+    pub fn with_defaults(
+        mutations: Vec<Mutation>,
+        primary: Vec<u8>,
+        start_ts: TimeStamp,
+    ) -> Command {
+        Prewrite::new(
+            mutations,
+            primary,
+            start_ts,
+            0,
+            false,
+            0,
+            TimeStamp::default(),
+            Context::default(),
+        )
+    }
+
+    #[cfg(test)]
+    pub fn with_lock_ttl(
+        mutations: Vec<Mutation>,
+        primary: Vec<u8>,
+        start_ts: TimeStamp,
+        lock_ttl: u64,
+    ) -> Command {
+        Prewrite::new(
+            mutations,
+            primary,
+            start_ts,
+            lock_ttl,
+            false,
+            0,
+            TimeStamp::default(),
+            Context::default(),
+        )
+    }
+
+    pub fn with_context(
```

Comment:
Seems this function is only used in test, can we add `[cfg(test)]`?

Reply:
No, it is used by the test crates and `cfg(test)` is not propagated upstream.